### PR TITLE
[TASK] Init commit for v12 branch

### PR DESCRIPTION
**Pull request by ayacoo**
TODO:

- Clarify open points (TYPO3 version support, ElementInfo etc.)

### CODE REVIEW

**Comment by ayacoo** [comment]
There will be a few things to discuss. First, the PR as a draft, so that there is no duplication of work. More to follow

File: composer.json [hunk 1]
```diff
@@ -1,16 +1,11 @@
 {
     "name": "b13/online-media-updater",
+	"version": "2.0.0",
```
Comment (bmack) [inline composer.json:3]: any reason why you add the "version"?

Comment (ayacoo) [reply composer.json:3]: I needed this for my local Dev environment. Will be switched again. Remember: This is a draft, I'm still developing ;-)

File: composer.json [hunk 2]
```diff
@@ -1,16 +1,11 @@
...
     "description": "Update YouTube/Vimeo preview images",
     "type": "typo3-cms-extension",
     "license": [
         "GPL-2.0-or-later"
     ],
-    "require": {
```
[the remark this answers is not in the document]
Comment (ayacoo) [reply composer.json:8]: Of course. I threw everything out for the time being. In principle, it also needs to be clarified how support is to continue. I'm not sure yet whether we can support v11 and v12 with this status. I'd actually like to make a cut and start with v12 + 8.1, but of course that's up to you.

**Comment by ayacoo** [comment]
[Rebuild typeRendering hook](https://github.com/b13/online-media-updater/pull/8/commits/dec61b9df3964a00338a86b1808d481700044678)

**Problems after Streamline of the ElementInformationController**
- There is no init function any more. Therefore, I had to add setter and getter to the core so that the hook would run again. Otherwise I can't access the protected values of the element.
- I have removed "our" AccessCheck as this is what the controller does in the mainAction and then calls the hook, if it even gets that far.
- In the core, a response interface is actually needed for the mainAction. I did not manage to overwrite the paths, it always wanted to use EXT:backend. Therefore, I rewrote my own function with return string and StandAloneView.
- The Constructor of ElementInformationController in v12 now needs 4 values and no longer only 3

**Pull request closed without merging**